code/lib/manager-api/src/modules/stories.ts

```diff
@@ -61,11 +62,14 @@ type ViewMode = 'story' | 'info' | 'settings' | string | undefined;
 type StoryUpdate = Partial<
   Pick<API_StoryEntry, 'prepared' | 'parameters' | 'initialArgs' | 'argTypes' | 'args'>
 >;
+type Status = Record<StoryId, Record<string, 'loading' | 'ready' | 'error' | 'warn' | 'unknown'>>;
```

Comment:
I feel like in general for "async" work like statuses, the status names are:

- pending (you have "loading")
- success (you have "ready"?)
- error
- warn
- unknown

that would be my recommendation.

Reply:
I like @JReinhold 's recommendation